remote/webby/webby_nn_aarch64_socket_bridge.h

```diff
@@ -0,0 +1,66 @@
+/************************************************************************************
```

Comment:
Note: this bridge is because the signatures in the architecture are different than in the generic includes. There are additional complications with different architecture include files contain different sets of signatures than the generic or cross platform ones. Note: this only effects the socket.h (and related files primarily), other include files are the same.